features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/attachments/preview/AttachmentsPreviewStateProvider.kt

```diff
@@ -10,22 +10,16 @@ package io.element.android.features.messages.impl.attachments.preview
 import androidx.compose.ui.tooling.preview.PreviewParameterProvider
 import androidx.core.net.toUri
 import io.element.android.features.messages.impl.attachments.Attachment
+import io.element.android.libraries.mediaviewer.api.MediaInfo
+import io.element.android.libraries.mediaviewer.api.anImageMediaInfo
 import io.element.android.libraries.mediaviewer.api.local.LocalMedia
-import io.element.android.libraries.mediaviewer.api.local.MediaInfo
-import io.element.android.libraries.mediaviewer.api.local.aVideoMediaInfo
-import io.element.android.libraries.mediaviewer.api.local.anApkMediaInfo
-import io.element.android.libraries.mediaviewer.api.local.anAudioMediaInfo
-import io.element.android.libraries.mediaviewer.api.local.anImageMediaInfo
 import io.element.android.libraries.textcomposer.model.TextEditorState
 import io.element.android.libraries.textcomposer.model.aTextEditorStateMarkdown
 
 open class AttachmentsPreviewStateProvider : PreviewParameterProvider<AttachmentsPreviewState> {
     override val values: Sequence<AttachmentsPreviewState>
         get() = sequenceOf(
             anAttachmentsPreviewState(),
-            anAttachmentsPreviewState(mediaInfo = aVideoMediaInfo()),
-            anAttachmentsPreviewState(mediaInfo = anAudioMediaInfo()),
-            anAttachmentsPreviewState(mediaInfo = anApkMediaInfo()),
```

Comment:
Since the content is not the real content, there is no need to have multiple identical previews.